.clangd.example

```diff
@@ -0,0 +1,35 @@
+# Global config for all files
+CompileFlags:
+  Add: [
+    # Remove all built-in definitions
+    "-undef",
+
+    # Indicator for things which shouldn't be processed in IDEs
+    "-DDECOMP_IDE_FLAG",
```

Comment:
I think the general case would be to use `#ifndef __MWERKS__` for this. I'm assuming this is because of the `clangd_compiler_defines` option defining `__MWERKS__`?

Reply:
I did *not* intend to leave that define there lol, forgot to remove it before committing.